decompressed/base/etc/init.d/rootdevice

```diff
@@ -330,7 +346,7 @@ debug() {
 boot() {
   ubus send fwupgrade '{"state":"upgrading"}'
   logo_project
-  check_full_rootfs
+  check_full_overlay
   check_root
```

Comment:
Dopo check_root ci metterei una pausa di sicurezza, direi almeno un minuto, così da poter intervenire e fermare quello che avviene dopo nel caso.

Reply:
cosa pensi che potrebbe accadere? conta che il check_root dovrebbe essere pensata per essere runnata ad ogni boot (e non dovrebbe far niente dato che dovrebbe trovarsi tutto già rootato)

Reply:
Accade che parte l'eventuale migrazione in OBP e successiva installazione della mod, e in caso di problemi (come ad esempio l'out-of-memory che hai considerato) finiresti in bootloop.

Reply:
si ma conta che siamo in uno stato davvero avanzato qui, siamo a circa 2minuti dopo che hai accesso seriale e circa 1minuto che probabile ti si attiva anche SSH, aggiungici il tempo computazionale, credo basti già tutto questo wait "fisiologico" se serve intervenire in parallelo

Reply:
ma questo tempo fisiologico di cui tu parli è dovuto all'estrazione dell'intero pacchetto, e al fatto che stai considerando la prima installazione, mentre questa funzione verrà comuqnue eseguita ogni volta al boot in tempi molto più corti

Reply:
no i tempi di cui parlo sono con gui già estratta ed installata e credo mi son tenuto anche stretto

altrimenti dovrei aggiungere qualche accrocchio per evitare che il delay si presenta anche su prima installazione lanciata da SSH